src/hosted-buttons/utils.js

```diff
@@ -289,41 +327,96 @@ export function getButtonColor(
+};
 
-  buttonContainer.setAttribute(
-    "style",
-    `display: flex; flex-wrap: nowrap; gap: 16px; max-width: 750px; flex-direction: ${flexDirection}`
+export const getDefaultButton = ({
+  buttonPreferences,
+  eligibleFundingMethods,
+}: HostedButtonPreferences): ?string => {
+  // Return first eligible funding method that is not specified in button preferences
+  return eligibleFundingMethods.find(
+    (fundingMethod) => !buttonPreferences.includes(fundingMethod)
   );
+};
 
-  const primaryButton = document.createElement("div");
-  primaryButton.setAttribute("id", `ncp-primary-button`);
-  primaryButton.setAttribute("style", "flex-grow: 1");
+export const renderStandaloneButton = ({
+  fundingSource,
+  preferences,
+  buttonContainerId,
+  buttonOptions,
+}: RenderStandaloneButtonProps): ZalgoPromise<void> | void => {
+  const Buttons = getButtonsComponent();
 
-  const secondaryButton = document.createElement("div");
-  secondaryButton.setAttribute("id", `ncp-secondary-button`);
-  secondaryButton.setAttribute("style", "flex-grow: 1");
+  const { buttonPreferences, eligibleFundingMethods } = preferences;
+  const { style } = buttonOptions;
 
-  buttonContainer.appendChild(primaryButton);
-  buttonContainer.appendChild(secondaryButton);
+  const isDefault = fundingSource === "default";
 
-  elm?.appendChild(buttonContainer);
-}
+  const fundingSourceName = isDefault
+    ? getDefaultButton({ buttonPreferences, eligibleFundingMethods })
+    : fundingSource;
+
+  // If getDefaultButton doesn't return a button, just return void
+  if (!fundingSourceName) {
+    return;
+  }
+
+  // $FlowFixMe
+  const standaloneButton = Buttons({
+    ...buttonOptions,
+    fundingSource: fundingSourceName,
+    style: {
+      ...style,
+      // $FlowFixMe
+      color: getButtonColor(style.color, fundingSourceName),
+    },
+  });
+
+  if (standaloneButton.isEligible()) {
+    return standaloneButton.render(`#${buttonContainerId}`);
+  }
+
+  getLogger().error(`ncps_standalone_${fundingSourceName}_ineligible`);
+
+  // If the funding source is explicitly defined, but is not eligible,
+  // don't render anything in it's place
+  if (!isDefault) {
+    return;
+  }
+
+  // Filter out ineligible funding method and everything before
+  const indexOfFundingSource =
+    eligibleFundingMethods.indexOf(fundingSourceName);
+  const filteredEligibleFundingMethods = eligibleFundingMethods.filter(
+    (_, index) => index > indexOfFundingSource
+  );
+
+  // If no eligible funding methods remain, return nothing
+  if (!filteredEligibleFundingMethods.length) {
+    return;
+  }
+
+  // Recursively call renderStandaloneButton to attempt to render the next eligible button.
+  return renderStandaloneButton({
+    fundingSource: filteredEligibleFundingMethods[0],
+    preferences: {
+      buttonPreferences,
+      eligibleFundingMethods: filteredEligibleFundingMethods,
+    },
+    buttonContainerId,
+    buttonOptions,
+  });
+};
```

Comment:
This is where the action happens.   Left a bunch of comments here just to help explain the logic, so hopefully that helps 😄